lib/bolt/config/src/ns.rs

```diff
@@ -557,6 +557,8 @@ pub struct Rivet {
 	pub cdn: Cdn,
 	#[serde(default)]
 	pub billing: Option<RivetBilling>,
+	#[serde(default)]
+	pub host_networking: bool,
```

Comment:
nit: this is going to lead to a messy config. can we move this in to a subsection like `rivet.matchmaker.enable_host_networking`?